records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -48,4 +52,11 @@ def move(self) -> MoveResult:
             driver = self.tbl.db_driver(db)
             schema_obj = self.fileobjs_source.records_schema
             schema_sql = self.schema_sql_for_load(schema_obj, self.records_format, driver)
-        return prep_and_load(self.tbl, self.prep, schema_sql, self.load, self.reset_before_reload)
+            loader_from_fileobj = driver.loader_from_fileobj()
+            # TODO: explain this one
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker